src/task.py

```diff
@@ -99,15 +99,39 @@ class HIIOSMRasterize(HIITask):
     def __init__(self, *args, **kwargs):
         super().__init__(self, *args, **kwargs)
 
-        self._args = kwargs
-
-        self.osmium_config = self._args.get("osmium_config")
-        self.process_roads = not self._args.get("no_roads")
-        _extent = self._args.get("extent")
+        self.osm_file = kwargs.get("osm_file")
+        self.osm_url = (
+            kwargs.get("osm_url")
+            or os.environ["OSM_DATA_SOURCE"]
+            or "https://ftp.fau.de/osm-planet/pbf/planet-latest.osm.pbf"
```

Comment:
Not sure if we should hard code the url in

Reply:
Yeah... but how can I avoid, since I can't specify in argparse if we want env vars to take effect? This is the same pattern for all the params.